CNV-38593: RN: Known issue Remove invalid annotation from "OVN Kubernetes secondary localnet network" (#71966)
openshift/openshift-docs: virt/release_notes/virt-4-15-release-notes.adoc

* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://issues.redhat.com/browse/CNV-33834[*CNV-33834*])
** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].

[discrete]
[id="virt-4-15-ki-networking"]
==== Networking
//CNV-38593: RN: Known issue Remove invalid annotation from "OVN Kubernetes secondary localnet network"

> * The invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears during the creation of a Network Access Device (NAD) with the `OVN Kubernetes secondary localnet network` network type. This annotation prevents the starting of the VM.

Reviewer: I am wondering if backticks should be applied to "OVN Kubernetes secondary localnet network" or not - Is this network type a part of a web console? In that case, backticks might not go here.

Author: Fixed

> ** As a workaround, remove the annotation.

Reviewer: ```suggestion
** As a workaround, remove the `k8s.v1.cni.cncf.io/resourceName: openshift.io/` annotation.
```
Totally up to you to decide!

Author: Left as is

* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://issues.redhat.com/browse/CNV-33834[*CNV-33834*])
** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].

[discrete]

> [id="virt-4-15-ki-networking"]

Reviewer 2: I think this is a web console bug

==== Networking
//CNV-38593: RN: Known issue Remove invalid annotation from "OVN Kubernetes secondary localnet network"

> * The invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears during the creation of a Network Access Device (NAD) with the *OVN Kubernetes secondary localnet network* network type. This annotation prevents the starting of the VM.

Reviewer 2: ```suggestion
* When creating a network attachment definition (NAD) for an OVN-Kubernetes localnet topology by using the web console, the invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears. This annotation prevents the starting of the VM.
```

Author: I looked at this in the console, and the **OVN Kubernetes secondary localnet network** appears at the  network type. Does it need to change?

Author: ![OVN Kubernetes network types](https://github.com/openshift/openshift-docs/assets/45106934/2ef7427e-4755-4bd9-aaa0-2b6c58f64b14)

Reviewer 2: Here is the web console procedure for creating a localnet NAD: https://docs.openshift.com/container-platform/4.15/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.html#virt-creating-nad-localnet-console_virt-connecting-vm-to-ovn-secondary-network. 
If you want to refer to the OVN-Kubernetes secondary network in a general way, remove the bold formatting. Otherwise, IMO, you will need to provide more context about where this field is in the UI.

//4.15 still unresolved
* When adding a virtual Trusted Platform Module (vTPM) device to a Windows VM, the BitLocker Drive Encryption system check passes even if the vTPM device is not persistent. This is because a vTPM device that is not persistent stores and recovers encryption keys using ephemeral storage for the lifetime of the `virt-launcher` pod. When the VM migrates or is shut down and restarts, the vTPM data is lost. (link:https://issues.redhat.com/browse/CNV-36448[*CNV-36448*])

//4.15 still unresolved
* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.

//4.15 Keep this per Stu, ETA for fixing this issue is 4.18.

> * With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled {op-system-base-full} 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected.

Reviewer: ```suggestion
* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS Extended Master Secret (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled {op-system-base-full} 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected.
```

Reviewer: Note: This is not in the scope of this PR. Up to you!
